packages/@okta/vuepress-site/docs/guides/submit-app/main/openidconnect/submit.md

```diff
@@ -15,6 +15,7 @@
   If you're using a per tenant design, include the variable names that you created. For example:
   * https://`${app.variableName}`.okta.com
   * https://okta-`${app.variableName}`.com
+  * `${app.variableName}`
```

Comment:
Is this valid to not have an `https://` at the start?

Reply:
Yes - this is the point of this change; the variable can now include http:// | https:// as part of the variable, which was requested by customers. I'll check with him if we should also use `${app.variableName}`/route here to better indicate this. The release note will point out that the variables for dynamic redirect URIs can now include this

Reply:
@IanHakes-okta for context: OPS asked us to allow a full URL as variable, which makes something like Redirect URL or ACS URL etc validation much simpler. https/.com are no longer required -- so long as a variable exists -- in order to allow customers to utilize more globally useful variables such as ${org.baseUrl}; I think we will commonly see these inside of a dynamic url input like: ${org.baseURL}/login etc.  Unfortunately, this doesn't work very well along side the previous examples.

Reply:
Maybe we need to add some text here to explain that?

> If you're using a per tenant design, include the variable names that you created. You can use a full URL as a variable. This enables you to use more globally useful variables such as `org.baseURL`. The `https://` and `.com` values are not strictly required.
 
 For example:
(etc.)